src/borg/testsuite/nonces.py

```diff
@@ -10,17 +10,6 @@
 from .. import nonces  # for monkey patching NONCE_SPACE_RESERVATION
 
 
-@pytest.fixture(autouse=True)
-def clean_env(monkeypatch):
-    # Workaround for some tests (testsuite/archiver) polluting the environment
-    monkeypatch.delenv('BORG_PASSPHRASE', False)
-
-
-@pytest.fixture(autouse=True)
-def nonce_dir(tmpdir_factory, monkeypatch):
-    monkeypatch.setenv('XDG_CONFIG_HOME', tmpdir_factory.mktemp('xdg-config-home'))
```

Comment:
btw one can actually import fixtures like anything else, it works like you'd expect.